Backpack-SwiftUI/PageIndicator/Classes/BPKPageIndicator.swift

```diff
@@ -0,0 +1,58 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import SwiftUI
+
+public struct BPKPageIndicator: View {
+    public enum Variant {
+        case `default`, overImage
+    }
+    
+    //    Switches between default and overImage style
+    public let variant: Variant
+    
+    //    This sets the current highlighted indicator
+    @State public var currentIndex: Int
```

Comment:
I think this and the one below should be just `public let` instead of `@State`, if you need binding behaviour, then this should be a `@Binding` and have the client of the component hold it's state, not this view itself